po/cs.po

```diff
@@ -87,74 +87,79 @@ msgstr "Šířka okraje"
 
 #: src/preferences/pages/general.ui:57 src/preferences/pages/general.ui:65
 #, fuzzy
-msgid "Border color"
+msgid "Focused border color"
 msgstr "Barva okraje"
 
-#: src/preferences/pages/general.ui:74
+#: src/preferences/pages/general.ui:74 src/preferences/pages/general.ui:82
+#, fuzzy
+msgid "Unfocused border color"
+msgstr "Barva okraje"
```

Comment:
Why is this the same as the string for the focused border color? This applies to other potfiles as well